mypy.ini

```diff
@@ -4,6 +4,23 @@ mypy_path = ./app,./app/stubs
 plugins =
     mypy_django_plugin.main
 explicit_package_bases = True
+exclude = ^.*[/\\]test_.*
+
+# add some additional checks, but not quite strict
+strict = True
+disallow_untyped_calls = False
+ignore_missing_imports = True
+implicit_reexport = true
```

Comment:
This is already included in strict mode, see [the mypy.ini equivalent to `--strict`](https://mypy.readthedocs.io/en/stable/existing_code.html#getting-to-strict). Can be removed here.

Reply:
The `implicit_reexport`? It's not (the other also not)

Reply:
The reason for allowing implicit reexports is to be able to directly use the import from another file, as done by you with `TranslationsSchema` in the `provider.api`. I don't see a value of forcing a user to not reuse this import...

Reply:
Oh my bad, you are right: I mixed it up with `no_implicit_reexport`. :hand_over_mouth:

Fine with me to keep it as it is then. I don't see why we should be forced to list exported symbols in the `__all__` attribute. Didn't even know that exists tbh.

Reply:
Or do something like `from xxx import yyy as yyy`, which is a bit pointless